app/categories/forms.py

```diff
@@ -23,11 +24,48 @@ class QuestionForm(Form):
         widget=CategoryRadioInput(
             show_divider=False
         ),  # Uses our override class to support setting custom CSS on the label title
-        validators=[InputRequired(message="Validation failed message")],
+        validators=[InputRequired(message=_("Validation failed message"))],
         choices=[
-            ("yes", "Yes"),
-            ("no", "No"),
+            ("yes", _("Yes")),
+            ("no", _("No")),
         ],
     )
 
-    submit = SubmitField("Continue", widget=GovSubmitInput())
+    submit = SubmitField(_("Continue"), widget=GovSubmitInput())
+
+    def __init__(self, *args, **kwargs):
+        if "category" in kwargs:
+            self.category = kwargs.pop("category")
+        super().__init__(*args, **kwargs)
+
+
+class SafeguardingQuestionForm(QuestionForm):
+    """Base form for question pages with configurable routing based on answers."""
```

Comment:
Can these duplicate comments be removed please, just to keep this class clean.

Reply:
Removed